evalai/utils/config.py

```diff
@@ -14,7 +14,7 @@
 
 AUTH_TOKEN_PATH = os.path.join(AUTH_TOKEN_DIR, AUTH_TOKEN_FILE_NAME)
 
-API_HOST_URL = os.environ.get("EVALAI_API_URL", "https://evalapi.cloudcv.org")
+API_HOST_URL = os.environ.get("EVALAI_API_URL", "http://127.0.0.1:8000")
```

Comment:
If we need to test for the EvalAI server on the production branch which is hosted, we need to use the host as `https://evalapi.cloudcv.org`. `http://127.0.0.1:8000` is used for the local EvalAI server.

Reply:
Yes. But making tests against the live server presented some problems as:
* An auth token would be required. Exposing an auth token would have security risks.
* The production database would have to be modified. And if we add tests for example, for making submissions, that would create a lot of duplicates on the in-use database if the test is run multiple times.

So as a walkaround, here the local EvalAI server is started from the production branch itself.
```
git clone --branch production https://github.com/Cloud-CV/EvalAI.git .travis-ci_tmp_evalai && cd .travis-ci_tmp_evalai
docker-compose up --build
```
So I tried to achieve the same thing, but with allowing more freedom over the data. 
I have included an analysis of both approaches in the report: https://github.com/nikochiko/evalai-cli/blob/test-integration-with-server/tests/integration/README.md